build.sbt

```diff
@@ -80,7 +80,13 @@ lazy val `surge-engine-command-javadsl` =
 lazy val `surge-engine-multilanguage` =
   (project in file("modules/multilanguage"))
     .dependsOn(`surge-engine-command-scaladsl`)
-    .settings(libraryDependencies ++= Seq(Akka.slf4j, logback, slf4jApi))
+    .settings(libraryDependencies ++= Seq(Akka.slf4j, logback, slf4jApi,
+      "com.typesafe.akka" %% "akka-protobuf-v3" % "2.6.16",
+      "com.typesafe.akka" %% "akka-pki" % "2.6.16",
+      "com.typesafe.akka" %% "akka-discovery" % "2.6.16",
```

Comment:
It doesn't look like these are actually used, but if so can you pull them into `project/Dependencies.scala` and version them with that same `Akka.version` variable to keep all of those consistent more easily?